randomiser/fill.js

```diff
@@ -0,0 +1,12 @@
+const fillRandomiserText = (ctx, imageCtx, height, width) => text => {
```

Comment:
Improve/test randomisation (potentially use some kind of sin/cos randomisation or other options)